vignettes/getting_started.Rmd

```diff
@@ -14,11 +14,12 @@ knitr::opts_chunk$set(
 )
 ```
 
-Were do you want to set-up `checklist`?
+Where do you want to set-up `checklist`?
 
 1. I want to create a [new R package](#create-package) from scratch.
 1. I want to use or update `checklist` in an [existing R package](#setup-package).
```

Comment:
The enumerations do not seem to add up in this document. It alsways says "1."

Reply:
`1. ` just indicates a numbered item. The actual numbering will be done when rendering the document. So no need to number the items yourself ;-)